### PR TITLE
Delete appointment

### PR DESCRIPTION
**Pull request by Bryan-Goh**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -104,17 +104,17 @@ There should not be any parameters after “list”. e.g. `list 123` will not re
 
 Deletes the appointment at the specified index of the specified patient.
 
-Format: `delete_appt INDEX /of T03XXXXXE`
+Format: `delete_appt INDEX /appt INDEX`
 
-* The command is case-sensitive. E.g. Delete appt INDEX /of T03XXXXXE will not work
+* The command is case-sensitive. E.g. delete_appt INDEX /appt 3 will not work
```
Comment (simbayippy) [inline docs/UserGuide.md:109]: Should this be Delete_appt?

**Review by tohpinren — CHANGES_REQUESTED** [review]
Please add newline in following files:
image

File: docs/UserGuide.md (hunk 1 above)
Comment (Bryan-Goh) [reply docs/UserGuide.md:109]: should be delete_appt

Comment (simbayippy) [reply docs/UserGuide.md:109]: the negative and positive formats appear to be the same, both of delete_appt

**Review by tohpinren — APPROVED** [review]

**Pull request merged**